content/src/content/jcr_root/apps/core/wcm/components/page/v3/page/headlibs.html

```diff
@@ -23,6 +23,8 @@
         ]}"></sly>
     <sly data-sly-include="/libs/cq/cloudserviceconfigs/components/servicelibs/servicelibs.jsp"></sly>
     <sly data-sly-test="${hasCloudconfigSupport}" data-sly-resource="${'cloudconfig-header' @ resourceType='cq/cloudconfig/components/scripttags/header'}"></sly>
+    <sly data-sly-test="${hasImageAutoSizeSupport}"
+         data-sly-call="${clientlib.js @ categories='core.wcm.components.commons.site.image.auto-size'}"></sly>
```

Comment:
As said before it would make more sense to let the image load the clientlib only on pages that actually have an autosized image on it. 

Load it async and deferred